static/src/js/actions/notifications.js

```diff
@@ -0,0 +1,163 @@
+import {
+  PUSHING_NOTIFICATION,
+  NOTIFICATION_PUSHED,
+  NOTIFICATION_DISMISSED
+} from '../constants/actionTypes'
+import {
+  toastPlacement,
+  toastType
+} from '../constants/enums'
+import { removeError } from './errors'
+
+const notificationDefaults = {
+  appearance: toastType.info,
```

Comment:
We'd prefer to just provide the library with strings that it documents, this particular enum isn't necessary as we'll provide this value each time a toast is created.